src/ucs/debug/memtrack.c

```diff
@@ -251,10 +263,18 @@ void ucs_memtrack_releasing(void* ptr)
     ucs_memtrack_entry_update(entry, -size);
     ucs_memtrack_entry_update(&ucs_memtrack_context.total, -size);
 
-out_unlock:
     pthread_mutex_unlock(&ucs_memtrack_context.lock);
 }
 
+static UCS_F_ALWAYS_INLINE void ucs_memtrack_releasing_internal(void *ptr)
+{
+    if (!ucs_memtrack_is_enabled()) {
```

Comment:
in master branch, ucs_malloc and co. code was changed to call ucs_memtrack_allocated_internal

Reply:
yes, fixed